bundles/org.eclipse.emfcloud.modelserver.edit/src/org/eclipse/emfcloud/modelserver/edit/util/CommandUtil.java

```diff
@@ -102,4 +117,200 @@ public static EObject createProxy(final EClass eClass, final String uri) {
       ((InternalEObject) eObject).eSetProxyURI(URI.createURI(uri));
       return eObject;
    }
+
+   public static Optional<CCommandExecutionResult> aggregateExecutionResults(
```

Comment:
Could you please elaborate what the purpose of this method is? It does not seem to be used anywhere.

Reply:
It is used with Transactions. When a transaction is open, we can execute multiple commands. Each command will generate a single CommandExecutionResult; but notifications will only be sent to listeners when the transaction is closed. At this point, we need to aggregate all available CommandExecutionResults into one. If it's not used (yet), then maybe a downstream project needs it (It's a public API after all). Maybe @cdamus can give more accurate info about this.

Reply:
It is indeed unused, almost certainly a vestige of some iteration of the code before this state. I'm happy to remove it.